src/mon/OSDMonitor.cc

```diff
@@ -644,7 +644,7 @@ class OSDUtilizationDumper : public CrushTreeDumper::Dumper<F> {
     if (average_util)
       var = util / average_util;
 
-    size_t num_pgs = pgm->get_num_pg_by_osd(qi.id);
+    size_t num_pgs = qi.is_bucket() ? 0 : pgm->get_num_pg_by_osd(qi.id);
```

Comment:
yes, pgm->get_num_pg_by_osd() always returns 0 if qi is a bucket. Therefor, it needn't to call it for a bucket.